caikit_computer_vision/data_model/object_detection.py

```diff
@@ -24,8 +24,23 @@
 from caikit.interfaces.common.data_model import ProducerId
 import alog
 
+# Local
+from .image_segmentation import ObjectSegment
```

Comment:
I think I would have a bounding box in the segmentation task instead of having a segmentation box in the object detection task. The segmentation masks are more costly not every object detection datasets have mask information; but having a segmentation mask we usually get the box almost no additional cost.